projects/graph/social/social.py

```diff
@@ -46,14 +47,28 @@ def populateGraph(self, numUsers, avgFriendships):
         self.users = {}
         self.friendships = {}
         # Add users
+        # for i in range(numUsers):
+        #     self.addUser(i)
+        # # Create friendships
+        # for j in range(numUsers):
+        #     for k in range(random.randint(0, avgFriendships+1)):
+        #         r_int = random.randint(1, numUsers)
+        #         print("r_int", r_int)
+        #         self.addFriendship(j, r_int)
+
+        # Redone code
+        # adds all users based on the parameter above
```

Comment:
Good it's great that you leave the mistakes in your code. This will be extremely helpful for you when you go back to review since you will be able to access your prior thought process.